apps/mobile/src/components/home/home.tsx

```diff
@@ -14,7 +15,7 @@ export function Home() {
   return (
     <HomeLayout>
       <AccountWidget accounts={accounts.list} wallets={wallets.list} />
-      <Earn />
+      <TokenWidget tokens={getMockTokens()} totalBalance={mockTotalBalance.totalUsdBalance} />
```

Comment:
@kyranjamie - I'm realising I have an erratic naming convention here. 

Should I go with plural or singular?

- `AccountWidget` vs `AccountsWidget`
    - `widgets/account/` vs  ``widgets/account/`
- `TokenWidget` vs `TokensWidget`
    - `widgets/token/token-widget` vs  ``widgets/tokens/tokens-widget` 
    
    etc.

Reply:
I think I'd error on side of plural, or `AccountListWidget` if this component reflects data of many